locales/fr.json

```
@@ -60,11 +81,23 @@
     },
     "customer": {
         "information": "Informations client",
-        "payment_info": "Mes infos de paiement"
+        "payment_info": "Mes infos de paiement",
+        "payment_method_update": {
+            "title": "Changement du moyen de paiement",
+            "outstanding_payment_failed": {
+                "title": "Échec du paiement",
+                "description": "Votre mode de paiement a été mis à jour, mais nous n'avons pas été en mesure de débiter le solde impayé. Veuillez utiliser un autre moyen de paiement."
+            }
+        }
+    },
+    "customer_details": {
+        "title": "Détails du client"
     },
     "customer_dashboard": {
         "my_account": "Mon compte",
         "ordered_on": "Commandé le",
+        "next_billing_date": "Next billing date",
+        "price": "Price",

```

Yeah, we should add that, and after that it will be 👌 to merge!
```suggestion
        "next_billing_date": "Prochaine date de facturation",
        "price": "Prix",
```